src/System.Private.CoreLib/shared/System/Threading/ThreadPool.cs

```diff
@@ -885,98 +987,114 @@ internal static void PerformWaitOrTimerCallback(_ThreadPoolWaitOrTimerCallback h
     public static partial class ThreadPool
     {
         [CLSCompliant(false)]
-        public static RegisteredWaitHandle RegisterWaitForSingleObject(
+        public static RegisteredWaitHandle RegisterWaitForSingleObject(  // throws RegisterWaitException
```

Comment:
These `throws RegisterWaitException` comments are useless. As far as I can tell, there is no type called `RegisterWaitException`.

Reply:
Yep, that's from the CoreCLR code. I will remove them.